server/src/cloud/board.ts

```diff
@@ -73,6 +73,7 @@ export type EditableBoardAttributes = {
   timerUTCEndTime?: Date;
   expirationUTCTime?: Date;
   joinConfirmationRequired?: boolean;
+  voting?: "active" | "disabled";
```

Comment:
Aren't there 3 states: actice, disabled and e.g. concluded?

Reply:
Good point! But I guess the concluded state will be added after finishing #594 or #595.